client/src/helpers/stardust/transactionsHelper.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TransactionsHelper } from "./transactionsHelper";
+
+describe("Transaction helper", () => {
```

Comment:
Looks good mostly!
Maybe we can add a couple more tests with different length of "index strings", since your function allows for "any" length.
And then test that it works properly with bigger strings. After that i think it's fine.

Reply:
Tests added.